tests/test_pneumatics_simulator.py

```diff
@@ -59,8 +62,44 @@ async def create_evt_cmd_client(
             )
             assert simulator.cmd_evt_server.connected
             assert cmd_evt_client.connected
+            await self.verify_almost_all_events(client=cmd_evt_client)
             yield cmd_evt_client
 
+    @contextlib.asynccontextmanager
+    async def create_telemetry_client(
+        self, simulator: atpneumaticssimulator.PneumaticsSimulator
+    ) -> typing.AsyncGenerator[tcpip.Client, None]:
+        async with tcpip.Client(
+            host=simulator.telemetry_server.host,
+            port=simulator.telemetry_server.port,
+            log=self.log,
+            name="TelemetryClient",
+        ) as telemetry_client:
+            await asyncio.wait_for(
+                simulator.telemetry_server.connected_task, timeout=TIMEOUT
+            )
+            assert simulator.telemetry_server.connected
+            assert telemetry_client.connected
+            yield telemetry_client
+
+    async def verify_event(
+        self,
+        client: tcpip.Client,
+        evt_name: str,
+    ) -> None:
+        data = await client.read_json()
+        assert "id" in data
+        assert data["id"] == evt_name
+
+    async def verify_almost_all_events(self, client: tcpip.Client) -> None:
```

Comment:
Please consider a doc string -- especially one that explains what you mean by "almost".

Reply:
Thanks for catching that. It is supposed to be called `verify_all_events` because it literally verifies that all events have been emitted. I added a doc string too.